src/vkcapture.c

```diff
@@ -70,6 +70,8 @@ typedef struct {
     int buf_id;
     int client_id;
     struct capture_texture_data tdata;
+    bool draw_opaque;
```

Comment:
Let's change it to `allow_transparency` and make it false by default - same as Windows game capture plugin.